app/src/displays/related-values/related-values.vue

```diff
@@ -108,16 +108,19 @@ function getLinkForItem(item: any) {
 </template>
 
 <style lang="scss" scoped>
+$toggle-px: 6px;
+$toggle-py: 4px;
```

Comment:
I don't think we're using SCSS variables in other components.
For the sake of consistency, let's use native CSS `var`.